espnet2/bin/asr_inference.py

```diff
@@ -79,11 +81,27 @@ def __init__(
         nbest: int = 1,
         streaming: bool = False,
         enh_s2t_task: bool = False,
+        quantize_asr_model: bool = False,
+        quantize_lm: bool = False,
+        quantize_modules: List[str] = ["Linear"],
+        quantize_dtype: str = "qint8",
     ):
         assert check_argument_types()
 
         task = ASRTask if not enh_s2t_task else EnhS2TTask
 
+        if quantize_asr_model or quantize_lm:
```

Comment:
I think we can safely remove the condition. If I'm not mistaken, we are dropping torch < 1.6 soon?
We can keep it for this PR though.

Reply:
I think we can keep it in this PR since we haven't dropped it.
If we no longer support these lower versions of PyTorch, we can remove this part.
I remember ESPnet1 quantization also has these version checks (even older ones that are not supported now). We may want to remove that as well.